src/lib/schemas.js

```diff
@@ -3,12 +3,12 @@ import { z } from 'zod';
 export const componentsSchema = z.array(
 	z.object({
 		title: z.string(),
-		npm: z
-			.string()
-			.regex(/(@[\w-]+\/)?[\w-]+/)
-			.optional(),
+		npm: z.string().regex(/^(@[a-z0-9-]*\/)?[a-z0-9-]*$/),
```

Comment:
> The `"name"` field contains your package's name, and must be lowercase and one word, and may contain hyphens and underscores.

Source: https://docs.npmjs.com/creating-a-package-json-file

underscore (`_`) is missing in your regex